src/components/nav.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import reactCSS from 'reactcss'
+
+const styles = reactCSS({
+  'default': {
+    nav: {
+      	height: '100%',
+      	backgroundColor: '#07263b',
+      	textAlign: 'center',
+      	color: 'white',
+      	paddingBottom: '0',
+      	paddingTop: '0',
+      	
+    },
+    title: {
+    	backgroundColor: '#0b304a',
+    	paddingBottom: '15px',
+    	paddingTop: '15px',
+    	marginTop: '0px',
+    	borderBottom: '1px #0e4b51 solid',
+      fontFamily: 'Fira Sans Condensed',
+    },
+    location: {
+      fontFamily: 'Fira Sans Condensed',
+    }
+  }
+})
+
+
+class Nav extends Component {
```

Comment:
`Nav` doesn't make use of any life-cycle hooks or state, so this component should be refactored as a functional component.